eloquentjs/4_1_sum_of_a_range.js

```diff
@@ -0,0 +1,85 @@
+/**
+ * Returns the range of numbers from between a-b by incrementing by the value of step
+ * If the sign of step is positive it returns an array ranging from the lesser(inclusive) to the greater
+ * If the sign of step is negative is returns an array ranging from the greater(inclusive) to the lesser
```

Comment:
If you're going to allow a negative step I wouldn't do anything special with the arguments. The first arg should be start and the second arg should be stop and if start is smaller than stop an array of length zero should be returned.